### ISSUE TITLE
Unable to execute JMX on Non GUI in Ubuntu

### ISSUE THREAD

**Issue by bholsmita** [apache/jmeter#6061]
### Expected behavior

JMX to run on ubuntu on 
Jmeter 5.5 
Openjdk version "11.0.20" 2023-07-18
OpenJDK Runtime Environment (build 11.0.20+8-post-Ubuntu-1ubuntu120.04)
OpenJDK 64-Bit Server VM (build 11.0.20+8-post-Ubuntu-1ubuntu120.04, mixed mode, sharing)
javac 11.0.20

Machine(VM) Configuration:
 
NAME="Ubuntu"
VERSION="20.04.5 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.5 LTS"
VERSION_ID="20.04"
HOME_URL=https://www.ubuntu.com/
SUPPORT_URL=https://help.ubuntu.com/
BUG_REPORT_URL=https://bugs.launchpad.net/ubuntu/
PRIVACY_POLICY_URL=https://www.ubuntu.com/legal/terms-and-policies/privacy-policy
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
 

### Actual behavior

Error Seen:
[($User)@perf-test-instance-0:~/apache-jmeter-5.5/bin$](mailto:ven_sbhol@perf-test-instance-0:~/apache-jmeter-5.5/bin$) jmeter -n -t /home/($User)/ )/{$Test.jmx}
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.thoughtworks.xstream.core.util.Fields (file:/usr/share/java/xstream.jar) to field java.util.TreeMap.comparator
WARNING: Please consider reporting this to the maintainers of com.thoughtworks.xstream.core.util.Fields
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Error in NonGUIDriver java.lang.IllegalArgumentException: Problem loading XML from:'/home/($User)/{$Test.jmx}’, conversion error com.thoughtworks.xstream.converters.ConversionException:
---- Debugging information ----
cause-exception     : com.thoughtworks.xstream.converters.ConversionException
cause-message       :
first-jmeter-class  : org.apache.jmeter.save.converters.TestElementConverter.unmarshal(TestElementConverter.java:109)
class               : org.apache.jmeter.save.ScriptWrapper
required-type       : org.apache.jmeter.save.ScriptWrapper
converter-type      : org.apache.jmeter.save.ScriptWrapperConverter
path                : /jmeterTestPlan/hashTree/hashTree/hashTree[9]/ResultCollector/objProp/value/sentBytes
line number         : 5425
version             : 2.13.20180731

### Steps to reproduce the problem

- Write a JMX file
- Upload in Ubuntu
- Make sure system matches to config mentioned above(In Expected)
- Run "jmeter -n -t <testfile.jmx> "

### JMeter Version

5.5

### Java Version

11.0.20

### OS Version

Ubuntu "20.04.5 LTS (Focal Fossa)"

**Comment by bholsmita** [comment]
Yes those are data which i renamed **intentionally**  to save PII. I do have the valid data running when hitting those commands.

**Comment by FSchumacher** [comment]
OK, so I assume, that you can open the same file in GUI mode?
If so, can you look at that file at line number 5425? Is there (or a line above) something related to "sentBytes"?
Are there any more error messages in jmeter.log?
